content/en/docs/concepts/configuration/secret.md

```diff
@@ -672,6 +672,33 @@ A container using a Secret as a
 Secret updates.
 {{< /note >}}
 
+{{< feature-state for_k8s_version="v1.18" state="alpha" >}}
+
+For clusters that are extensively using secrets (having at least tens of thousands of unique
+Secret to Pod mounts), Kubernetes has an _Immutable Secrets/ConfigMaps_ feature. It allows you
+to mark individual Secrets/ConfigMaps as `Immutable`, which blocks any changes to their data
+and in return:
+- protects you from accidental bad updates that could cause applications outages
+- improves performance of your cluster by significantly reducing load on kube-apiserver, by
```

Comment:
Hello @wojtek-t ,
I wonder if a new commit would fix the deploy preview. Could you add a new commit?
line 681, Possible edit: `- protects you from accidental (or unwanted) updates
line 699, spelling, secret

/cc @VineethReddy02

Reply:
@wojtek-t , Would you rebase this PR? I am interested in viewing a preview of the page.
You may have picked up a previous version of the `zh` deployment.md file. The build error has been corrected.

Reply:
done